packages/manager/modules/nutanix/src/dashboard/component/licence-tile/licence-tile-legacy-features/template.html

```diff
@@ -0,0 +1,46 @@
+<oui-tile-definition
+    data-ng-repeat="license in $ctrl.licenses.getFeatures() | limitTo: ($ctrl.expand ? $ctrl.licenses.getFeatures().length : $ctrl.defaultLicenseLength) track by license.name"
+>
+    <oui-tile-term data-ng-bind=":: license.getFormatedName()"></oui-tile-term>
+    <oui-tile-description>
+        <span
+            class="oui-badge oui-badge_success"
+            data-ng-if="license.isEnabled() && license.name !== 'lifeCycleManager'"
+            data-translate="nutanix_cluster_licence_included"
+        ></span>
+        <span
+            data-ng-if="!license.isEnabled() && license.name !== 'files' && license.name !== 'objects'"
```

Comment:
I don't know if it's possible to put less conditional logic in html file. Maybe use a function in controller.